sdcm/logcollector.py

```diff
@@ -894,6 +894,9 @@ class MonitorLogCollector(LogCollector):
         FileLog(name='scylla_manager.log',
                 command='sudo journalctl -u scylla-manager.service --no-tail',
                 search_locally=True),
+        FileLog(name='manager_scylla_backend.log',
+                command='sudo journalctl -u scylla-server.service --no-tail',
```

Comment:
Don't we need jmx and other parts ? I.e. same as we do for regular scylla

Reply:
that is a good question, but after started, it does very little trouble, and the problem was during some tests, were the backend possibly had problems...
but yes, if not to hard, @ShlomiBalalis , please add all other backend logs, like we have for each db node

Reply:
The only other service that exists in the monitor is scylla jmx. I added it as well.